lib/ruby_lsp/event_emitter.rb

```diff
@@ -40,7 +40,7 @@ def emit_for_target(node)
       when YARP::CallNode
         @listeners[:on_call]&.each { |l| T.unsafe(l).on_call(node) }
       when YARP::ConstantPathNode
-        @listeners[:on_constant_path_node]&.each { |l| T.unsafe(l).on_constant_path_node(node) }
+        @listeners[:on_constant_path]&.each { |l| T.unsafe(l).on_constant_path(node) }
```

Comment:
I agree with the rename, but this event was already using YARP, which means some other request that was already migrated needs to be updated to match the new event name. It's very likely document symbol, but we need to double-check.

Reply:
Document symbol uses only `:on_constant_path_write` and `on_constant_write`:

https://github.com/Shopify/ruby-lsp/blob/yarp/lib/ruby_lsp/requests/document_symbol.rb

`on_constant_path_node` was introduced in the [Definition Request migration](https://github.com/Shopify/ruby-lsp/pull/864) but it looks like it wasn't actually necessary for that PR.

I verified that `test/requests/definition_expectations_test.rb` and `test/requests/document_symbol_expectations_test.rb` still pass.

Reply:
We definitely need that event in [definition](https://github.com/Shopify/ruby-lsp/blob/82463dc9c7ee9ca7992c2cd6f8fb24fb86d8bcb6/lib/ruby_lsp/requests/definition.rb#L50). It's possible we don't have a test for that scenario.

Don't trust the tests as perfect sources of truth because it's likely we missed other scenarios too. If you identify missing tests, feel free to push those directly to main, so that we avoid any regressions too.

Reply:
Definition was updated in https://github.com/Shopify/ruby-lsp/pull/878 but isn't yet in the YARP branch. I don't want to rebase the `yarp` branch right now, as it will affect the open PRs against that. Once everything is complete we can rebase.